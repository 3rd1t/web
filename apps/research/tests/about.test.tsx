import React from "react"
import { render } from "@testing-library/react"
import renderer from "react-test-renderer"
import About from "../pages/about"

describe(" About", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<About />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
