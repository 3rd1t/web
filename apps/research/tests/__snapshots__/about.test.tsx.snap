// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` About renders correctly 1`] = `
<section
  className="text-gray-700"
>
  <div
    className="container px-5 py-24 mx-auto"
  >
    <div
      className="flex flex-col justify-center w-full text-center"
    >
      <h1
        className="text-2xl tracking-widest text-gray-900"
      >
        OUR TEAM
      </h1>
    </div>
    <div
      className="flex flex-wrap justify-center mt-20 -m-4 sm:justify-start"
    >
      <div
        className="p-4 lg:w-1/2"
      >
        <div
          className="flex flex-col items-center h-full text-center sm:flex-row sm:justify-start sm:text-left"
        >
          <img
            alt="Nicolas Webersinke"
            className="flex-shrink-0 object-cover object-center w-40 h-40 rounded-full shadow-xs"
            src="https://imgflip.com/s/meme/Mocking-Spongebob.jpg"
          />
          <div
            className="flex flex-col sm:pl-8"
          >
            <h2
              className="text-lg font-medium text-gray-900"
            >
              Nicolas Webersinke
            </h2>
            <h3
              className="text-gray-500"
            >
              CEO!!11!
            </h3>
            <p
              className="max-w-xs mt-3 sm:max-w-none"
            >
              I don't mean to brag, but I put together a puzzle in one day and the box said 2-4 years.
            </p>
          </div>
        </div>
      </div>
      <div
        className="p-4 lg:w-1/2"
      >
        <div
          className="flex flex-col items-center h-full text-center sm:flex-row sm:justify-start sm:text-left"
        >
          <img
            alt="Kevin Kohler"
            className="flex-shrink-0 object-cover object-center w-40 h-40 rounded-full shadow-xs"
            src="https://i.kym-cdn.com/entries/icons/original/000/016/546/hidethepainharold.jpg"
          />
          <div
            className="flex flex-col sm:pl-8"
          >
            <h2
              className="text-lg font-medium text-gray-900"
            >
              Kevin Kohler
            </h2>
            <h3
              className="text-gray-500"
            >
              Ganz wichtig
            </h3>
            <p
              className="max-w-xs mt-3 sm:max-w-none"
            >
              I put my phone in airplane mode, but it's not flying.
            </p>
          </div>
        </div>
      </div>
      <div
        className="p-4 lg:w-1/2"
      >
        <div
          className="flex flex-col items-center h-full text-center sm:flex-row sm:justify-start sm:text-left"
        >
          <img
            alt="Andreas Thomas"
            className="flex-shrink-0 object-cover object-center w-40 h-40 rounded-full shadow-xs"
            src="https://i.kym-cdn.com/entries/icons/mobile/000/020/002/memeeman.jpg"
          />
          <div
            className="flex flex-col sm:pl-8"
          >
            <h2
              className="text-lg font-medium text-gray-900"
            >
              Andreas Thomas
            </h2>
            <h3
              className="text-gray-500"
            >
              Does things
            </h3>
            <p
              className="max-w-xs mt-3 sm:max-w-none"
            >
              Why is monday so far from friday, and friday so close to monday?
            </p>
          </div>
        </div>
      </div>
      <div
        className="p-4 lg:w-1/2"
      >
        <div
          className="flex flex-col items-center h-full text-center sm:flex-row sm:justify-start sm:text-left"
        >
          <img
            alt="Mads Jordt"
            className="flex-shrink-0 object-cover object-center w-40 h-40 rounded-full shadow-xs"
            src="https://de.meming.world/images/de/thumb/2/29/Leere_Vorlage_HD_Pikachu.jpg/300px-Leere_Vorlage_HD_Pikachu.jpg"
          />
          <div
            className="flex flex-col sm:pl-8"
          >
            <h2
              className="text-lg font-medium text-gray-900"
            >
              Mads Jordt
            </h2>
            <h3
              className="text-gray-500"
            >
              Facility Manager
            </h3>
            <p
              className="max-w-xs mt-3 sm:max-w-none"
            >
              If you think your boss is stupid, remember: you wouldn't have a job if he was any smarter.
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>
`;
